src/solvers/refinement/string_functions.cpp

```diff
@@ -0,0 +1,19 @@
+/*******************************************************************\
+
+Module: Defines identifiers for string functions
+
+Author: Romain Brenguier
+
+Date:   September 2016
+
+\*******************************************************************/
+
+#include <solvers/refinement/string_functions.h>
+
+bool starts_with(irep_idt id, irep_idt prefix) {
+  std::string s = id2string(id);
+  std::string t = id2string(prefix);
+  for(int i = 0; i < t.length(); i++)
+    if(s[i] != t[i]) return false;
+  return true;
+}
```

Comment:
done in https://github.com/diffblue/cbmc/pull/341/commits/9efef8fd31d0b336b082ee7b754a0c6b2dd9d496